t/valgrind/valgrind-workload.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 echo FLUX_URI=$FLUX_URI
-TESTDIR=${SHARNESS_TEST_SRCDIR:-.}/valgrind/workload.d/*
 exitcode=0
 
-# Check for no workload:
-test -d ${TESTDIR} || exit $exitcode
+flux module remove -f sched-simple
+flux module load sched-fluxion-resource
```

Comment:
I think we need to `load-allowlist` to produce a resource graph conforming to Jobspect V1. 

```suggestion
flux module load sched-fluxion-resource
flux module load sched-fluxion-resource load-allowlist=node,core,gpu
```